packages/compass-aggregations/src/components/create-view-modal/create-view-modal.jsx

```diff
@@ -73,11 +72,12 @@ class CreateViewModal extends PureComponent {
         trackingId="create_view_modal"
         data-testid="create-view-modal"
       >
-        <ModalInput
-          id="create-view-name"
-          name="Enter a View Name"
+        <TextInput
+          data-testid="create-view-name"
           value={this.props.name || ''}
-          onChangeHandler={this.onNameChange}
+          onChange={this.onNameChange}
+          label="Name"
```

Comment:
We often have <Label> as a separate element or we jump through hoops to hide the label and use a placeholder or have some nearby heading act as a implicit label or sometimes we even use TextInput's builtin label like here.

I just standardised on using TextInput's label for these modals.